src/lib/radio/radio.ts

```diff
@@ -260,6 +275,9 @@ export class MdRadioButton implements OnInit {
   /** The 'aria-labelledby' attribute takes precedence as the element's text alternative. */
   @Input('aria-labelledby') ariaLabelledby: string;
 
+  /** Whether this radio is required. */
+  @Input('required') required: any = null;
```

Comment:
If the name of the input matches the name of the field, you don't need to specify it.